### ISSUE TITLE
quiz featuring a custom grader question precheck/check goes to custom HTML that immediately reverts back to original question page

### ISSUE THREAD

**Issue by kcleung** [trampgeek/moodle-qtype_coderunner#33]
In a quiz that has a custom grader question, when the user clicks "pre-check" or "check" in that custom grader question, the browser navigates to the custom HTML with the images and the custom result table, but after a fraction of seconds, the browser goes back to the original question page, leaving the user with no chance to see the check outcome.

To reproduce this bug:

* install Lubuntu 16.04 in a VM
* install moodle 3.2.2, or 3.1.4
* pull git moodle-qtype_coderunner into moodle/question/type/coderunner
* pull git moodle-qbehaviour_adaptive_adapted_for_coderunner into moodle/question/behaviour/adaptive_adapted_for_coderunner
* navigate to localhost/moodle in firefox in the VM
* create a course
* import the attached question that has a custom grader
* create a quiz
* add the imported question into the quiz
* attempt the quiz
* type in answer as instructed in the question
* click "precheck" or "check"

[question.zip](https://github.com/trampgeek/moodle-qtype_coderunner/files/866239/question.zip)

**Comment by kcleung** [comment]
I removed the embedded image from the toy custom grader question attached in the last message in this issue, but the behaviour stayed the same.

Then I proceeded and removed 'prologuehtml' and 'epiloguehtml' altogether from the response json from the question template code, leaving only 'testresults' and 'fraction' in the output json, so there should be no HTML elements generated by the question template, but the behaviour is still the same.

​So it looks like that image is not an issue, but it is Coderunner having a bug on displaying Precheck and check custom grader output.
